src/messages.js

```diff
@@ -110,6 +110,10 @@ export const identifyDisabled = function() {
   return 'identify() has no effect here; it must be called on the main client instance';
 };
 
+export const doNotTrackEnabled = function() {
+  return 'Not sending events to LaunchDarkly because Do Not Track is enabled.';
```

Comment:
Minor: let's rephrase this as:

```suggestion
  return 'LaunchDarkly events will not be sent because this browser's Do Not Track setting is enabled.';
```

Reply:
Fine by me!